docs/MAINTAIN.md

```diff
@@ -0,0 +1,16 @@
+# Supporting new families
+
+## Write device file
+
+a file <family>.cmake must be added to the folder cmake/stm32
+This file containes the differents devices and the regex used to parse them.
+It also give information on the RAM and CCRAM available
+
```

Comment:
```suggestion
Maintainer must check than Flash sizes correspond to usual suffix. Some families have exceptions.

```
+ add the suffix or hint file containing the sufix to size code

Reply:
Can you make a suggestion with all the changes you want here ?